Lab2/scripts/deployment.sh

```diff
@@ -19,6 +19,17 @@ while [[ "$#" -gt 0 ]]; do
     shift
 done
 
+# Check if workshop is running through EventEngine or not
+IS_RUNNING_IN_EVENT_ENGINE=false
+PREPROVISIONED_ADMIN_SITE=$(aws cloudformation list-exports --query "Exports[?Name=='Serverless-SaaS-AdminAppSite'].Value" --output text)
```

Comment:
will this not return true when we are re-running the script as part of Lab2?

Reply:
No, when we run workshop as a part of event engine we create cloud front & S3 buckets and export those resources values and the cloudformation command here is looking for exported values.
When we run workshop in our own AWS account in those cases when cloud front and S3 buckets are created we are just output those resources values and the cloudformation command we use will look for output values. In this case the exported values will be empty.